packages/shared/src/utils/config/config.ts

```diff
@@ -52,31 +52,30 @@ export const getAppId = () => {
     const current_domain = getCurrentProductionDomain() || '';
     window.localStorage.removeItem('config.platform'); // Remove config stored in localstorage if there's any.
     const platform = window.sessionStorage.getItem('config.platform');
-    const { is_pathname_bot, is_config_route_bot } = isBot();
+    const domain_app_id = domain_app_ids[current_domain as keyof typeof domain_app_ids];
+    const bot_app_id = isStaging() ? 19112 : 19111;
+    const set_app_id = isBot() ? bot_app_id : domain_app_id;
+
     // Added platform at the top since this should take precedence over the config_app_id
     if (platform && platform_app_ids[platform as keyof typeof platform_app_ids]) {
         app_id = platform_app_ids[platform as keyof typeof platform_app_ids];
-    } else if (config_app_id) {
+    }
+    if (config_app_id) {
         app_id = config_app_id;
-    } else if (user_app_id.length) {
+    }
```

Comment:
Here if we are removing the `else if` then it would break the precedence order, can you please check if the last one should get the higher precedence? If so then we can keep the `else if` structure and just put the bottom logic at the top.